packages/ui/src/composables/useNumericProp.ts

```diff
@@ -0,0 +1,19 @@
+import { computed, getCurrentInstance } from 'vue'
+
+/**
+ * @param key key of props
+ * @returns Numeric type of key attribute
+ */
+export const useNumericProp = (key: string) => {
+  const numeric = getCurrentInstance()!.props?.[key] as string | number | undefined
+
+  const numericComputed = computed(() => {
+    if (numeric === undefined) {
+      return numeric
+    }
+
+    return Number(numeric)
+  })
+
+  return { numericComputed }
```

Comment:
Let's return computed, so it is easy to rename it
```ts
const size = useNumericProp('size')
```
instead of

```ts
const { numericComputed: size } = useNumericProp('size')
```


```suggestion
  return numericComputed
```